apps/stats/management/commands/update_counts_from_file.py

```diff
@@ -27,6 +26,12 @@
                           """, re.VERBOSE)
 VALID_STATUSES = ["userDisabled,incompatible", "userEnabled", "Unknown",
                   "userDisabled", "userEnabled,incompatible"]
+VALID_APP_GUIDS = amo.APP_GUIDS.keys()
+APPVERSION_REGEX = re.compile(
+    r"""^[0-9]{1,3}                # Major version: 2, 35
+        \.[0-9]{1,3}([ab][0-9])?   # Minor version + alpha or beta: .0a1, .0b2
+        (\.[0-9]{1,3})?$           # Patch version: .1, .23
```

Comment:
There could *technically* be a 4th component, but we haven't done that since 2.0, so this is probably better.